public/app/types/alerting.ts

```diff
@@ -142,36 +142,37 @@ export interface NotificationChannelState {
   notificationChannel: any;
 }
 
-export interface IntegrationError {
-  lastError: string;
+export interface IntegrationStatus {
+  lastError?: null | string;
   lastNotify: string;
-  lastNotifyDuration: number;
+  lastNotifyDuration: string;
```

Comment:
I'm guessing some of these were typed incorrectly before? :D

Reply:
yes!, that's changed to string because of @konrad147 suggestion.